lib/Ohmbrewer_Sprouts.cpp

```diff
@@ -409,8 +410,10 @@ int Ohmbrewer::Sprouts::parseHeatingElementPins(char* params, std::list<int> &el
 
     // Verify that the pins are not in use
     elementPins.push_back(controlPin.toInt());
-    if(powerPin != NULL && powerPin.toInt() > -1) {
+    if(powerPin != NULL ) {
         elementPins.push_back(powerPin.toInt());
+    }else{
+        elementPins.push_back(-1);//powerPin not specified
```

Comment:
i had origionally done this wehn you wanted to change it so -1s where passed along but it isnt needed so im fixing it.